docs/sources/write/style-guide/word-list/index.md

```diff
@@ -112,7 +118,8 @@ Use this rather than _hamburger menu_ or _kebab menu_.
 
 ### no-collector
 
-Use this to refer to deployments that don't have a collector. Use this instead of _agentless_.
+Use this to refer to deployments that don't have a collector.
+Use this instead of _agentless_.
```

Comment:
```suggestion
<!-- vale Grafana.Agentless = NO -->

Use this to refer to deployments that don't have a collector.
Use this instead of _agentless_.

<!-- vale Grafana.Agentless = YES -->
```